### DISCUSSION TITLE
Traducción de la palabra "legacy"

### DISCUSSION THREAD

**Discussion by mateoguzmana** [reactjs/es.reactjs.org#678] [Q&A] [answered]
Al traducir la palabra "legacy", el termino que se usa actualmente es «heredado» (ejemplo: código heredado, API heredada).

Sin embargo cuando se lee la documentación, se siente un poco raro usar «heredado». No transmite lo mismo en mi opinión personal (depende de el caso, claro). [Hay un ejemplo de esto aquí](https://github.com/reactjs/es.react.dev/pull/660/files)

¿qué palabra podría usarse alternativamente?

**Comment by NGHdeveloper** [comment, marked as answer]
buen punto, para comenzar, en el caso del ejemplo no se debe traducir APIs sino "Las API" para los [acrónimos](https://www.rae.es/dpd/acr%C3%B3nimo#S1590507317464736095) y las [siglas](https://www.rae.es/dpd/sigla#S1590507322106881458) según RAE. La traducción hacia "heredado/a" correspondería a la palabra en inglés "inherited", por lo tanto no es una traducción literal
Alternativas para el ejemplo:

1. Las API React legadas (traducción literal)
2. Las API React antiguas
3. Las API React anteriores
4. Las API React existentes
5. Las API React obsoletas

El punto 5 de la lista es la idea mas aproximada para el contexto del caso de ejemplo, puesto que si observas todas las API reunidas en ese apartado tiende o a ser obsoletas o recomiendan migrar, pero el punto 1 sería la traducción correcta, puesto que React define "inherited" para las propiedades "heredadas" de JavaScript [(véase inherited)](https://github.com/search?q=repo%3Areactjs%2Freact.dev+inherited&type=code), y "legacy" para todo aquello "legado" por Reactjs [(véase legacy)](https://github.com/search?q=repo%3Areactjs%2Freact.dev+legacy&type=code).

**Reply by Byron2016** [reply]
La explicación es muy buena, pero en el [Glosario](https://github.com/reactjs/es.react.dev/wiki/Glosario) solo indica `heredado (código heredado, API heredada)`, propongo que se agreguen esos 5 ejemplos de acuerdo al contexto del documento original. 
¿Qué opinan?
